config/locales/en.yml

```
@@ -34,16 +34,15 @@ en:
                     <a href="https://www.railstutorial.org/contact">contact page</a>.
                   </p>
   base_title: Ruby on Rails Tutorial Sample App
-  footer_html: <small>
-                  The <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a>
-                  by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
-                </small>
-  footer1_html: <li><a href="http://news.railstutorial.org/">News</a></li>
-  header_html:  <li><%= link_to "Home",   "#" %></li>
-                <li><%= link_to "Help",   "#" %></li>
-                <li><%= link_to "Log in", "#" %></li>
-
-  footer2_html: <li><%= link_to "About",   '#' %></li>
-                <li><%= link_to "Contact", '#' %></li>
+  footer: The Ruby on Rails Tutorial by Michael Hartl
+  News: News
```

tên key in thường hết nha em
```suggestion
  news: News
```